### PR TITLE
Add minor formatting changes and text edits to README

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,21 +3,25 @@
...
 
-rx-jupyter is a Reactive wrapper around the Jupyter Server API.  What is this
-library useful for? It can help you query the services API on local and remote
-instances of Jupyter and integrate it seamlessly with RxJS's functional tooling.
+**rx-jupyter** is a Reactive wrapper around the [Jupyter Server API].
+
+## Why use rx-jupyter?
+
+**rx-jupyter** can help you query local and remote Jupyter Server instances
+using Jupyter's Services API. Also, **rx-jupyter** integrates responses
```
Comment (rgbkrk) [inline README.md]: Would this be

> Jupyter's Services APIs

File: README.md [hunk 2]
```diff
@@ -3,21 +3,25 @@
 [![Build Status](https://travis-ci.org/nteract/rx-jupyter.svg?branch=master)](https://travis-ci.org/nteract/rx-jupyter)
```
Comment (rgbkrk) [inline README.md]: Woo! Thanks for the PR.

**Comment by willingc** [comment]
How would you refer to this library if you were speaking the library name: 'R-X jupyter' or 'React jupyter' or 'Reactive jupyter'? 

Might be good to add.

File: README.md (hunk 1 above)
Comment (willingc) [inline README.md]: Why plural? Seriously.

**Comment by rgbkrk** [comment]
> How would you refer to this library if you were speaking the library name: 'R-X jupyter' or 'React jupyter' or 'Reactive jupyter'?

R-X jupyter

**Comment by rgbkrk** [comment]
and for fun I suppose:

> /ɑr ɛks ˈʤupətər/

:wink:

**Comment by rgbkrk** [comment]
Looks great, I went ahead and merged. If you've got more, we can keep on shipping it.

**Pull request merged**